descriptors/ModuleDescriptor-template.json

```diff
@@ -31,6 +31,16 @@
           "methods": [ "POST" ],
           "pathPattern": "/token/refresh",
           "permissionsRequired": [ "auth.signrefreshtoken" ]
+        },
+        {
+          "methods": [ "POST" ],
+          "pathPattern": "/token/logout",
+          "permissionsRequired" : [ ]
+        },
+        {
+          "methods": [ "DELETE" ],
+          "pathPattern": "/token/logout-all",
```

Comment:
For me this looks odd.

If there's POST /token/logout I would expect POST /token/logout-all.

https://datatracker.ietf.org/doc/html/rfc2616#section-9.7 "The DELETE method requests that the origin server delete the resource identified by the Request-URI."

Is the resource "logout-all" to be deleted?

Reply:
I agree that the POST to /token/logout is inconsistent with the DELETE to /token/logout-all. My goal was to use DELETE for both operations here like I did mod-login, but ended up not being able to make this happen due the the need for a body to /token/logout.

Let me change these to both be POST here. Let me know in the mod-login PR if  you'd like to have those be POST as well.

Reply:
Yes, use POST there as well.

Reply:
Ok will change mod-login to POST for the new logout endpoints there as well.